job_templates/ci_job.xml.em

```diff
@@ -133,6 +133,10 @@ if [ "$CI_USE_OPENSPLICE" = "true" ]; then
 fi
 if [ -n "${CI_ROS2_REPOS_URL+x}" ]; then
   export CI_ARGS="$CI_ARGS --repo-file-url $CI_ROS2_REPOS_URL"
+@[ if turtlebot_demo ]@
+else
+  export CI_ARGS="$CI_ARGS --repo-file-url @default_repos_url"
```

Comment:
Yeah, the readability there is pretty poor. That conditional will pop better when it's not awkwardly split by the template conditional.